docs/resources/Entitlement.md

```diff
@@ -94,6 +95,28 @@ Returns all entitlements for a given app, active and expired.
 ]
 ```
 
+## Get Entitlement % GET /applications/{application.id#DOS_RESOURCES_APPLICATION/application-object}/entitlements/{entitlement.id#DOCS_RESOURCES_ENTITLEMENT/entitlement-object}
+
+Returns an entitlement.
+
+```json
+{
+  "id": "1019653849998299136",
+  "sku_id": "1019475255913222144",
+  "application_id": "1019370614521200640",
+  "user_id": "771129655544643584",
+  "promotion_id": null,
+  "type": 8,
+  "deleted": false,
+  "gift_code_flags": 0,
+  "consumed": false,
+  "starts_at": "2022-09-14T17:00:18.704163+00:00",
+  "ends_at": "2022-10-14T17:00:18.704163+00:00",
+  "guild_id": "1015034326372454400",
+  "subscription_id": "1019653835926409216"
+}
```

Comment:
Some properties here don't appear to be on the entitlement object documented. Should they be removed?

Reply:
This is the same object as documented for the list entitlements endpoint.